src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function cyclingFunc(param) {
+    if (param !== undefined) {
+      sum += param;
+
+      return cyclingFunc;
+    } else {
```

Comment:
you don't need else here
```suggestion
    }
```